tox.ini

```diff
@@ -1,47 +1,56 @@
 [tox]
-minversion = 1.8.0
 envlist =
-    coverage-clean
-    py27-{111}
-    py34-{111,20}
-    py35-{111,20}
-    py36-{20,master}
-    coverage-report
+    {py36}-{django21,django22,django30,djangomaster}
```

Comment:
Are you sure you do not want to support python 3.5? 

> Django 2.1 supports Python 3.5, 3.6, and 3.7. Django 2.0 is the last version to support Python 3.4. We highly recommend and only officially support the latest release of each series.

https://docs.djangoproject.com/en/2.2/releases/2.1/

Reply:
Agreed, added Python 3.5 again.